reactor-core/src/main/java/reactor/core/publisher/FluxPublish.java

```diff
@@ -552,6 +553,7 @@ public Object scanUnsafe(Attr key) {
 			if (key == Attr.BUFFERED) return queue != null ? queue.size() : 0;
 			if (key == Attr.TERMINATED) return isTerminated();
 			if (key == Attr.CANCELLED) return s == Operators.cancelledSubscription();
+			if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
the two other inners should probably be covered too

Reply:
Fixed in commit [26552f6](https://github.com/reactor/reactor-core/pull/2123/commits/26552f67f2cfd7c10e59e60f3b905504c55b6e1d)